### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#420] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my crosh is disabled and I use lightspeed agent, can anyone help me unblock my Chromebook? ps javascript bookmarklets are blocked so is every setting in the Chromebook settings.

**Comment by obkduishinijf** [comment, marked as answer]
all i can say is good luck 🤷

**Reply by EmpoweredPhoenix** [reply]
sad

**Reply by Calamitou** [reply]
use a proxy

**Comment by Calamitou** [comment]
you got question #420!!! good job

**Reply by Aka-but-nice** [reply]
I wonder who got question #69

**Reply by Aka-but-nice** [reply]
smh there is not a question
